Unix/installbuilder/GNUmakefile

```diff
@@ -159,6 +159,13 @@ else #($(PF),Darwin)
 	# copy normal libmi.so for other platforms
 	cp $(OUTPUTDIR)/lib/libmi.so $(PACKAGE_DIR)
   endif #ifeq ($(PF),Linux)
+ 
+  # copy some providers for Linux/Unix platforms
+  mkdir -p $(PACKAGE_DIR)/test
+  cp $(OUTPUTDIR)/lib/libalert.so $(PACKAGE_DIR)/test
+  cp $(OUTPUTDIR)/lib/libfilesys.so $(PACKAGE_DIR)/test
+  cp $(OUTPUTDIR)/lib/libPresidentProvider.so $(PACKAGE_DIR)/test
+  cp $(OUTPUTDIR)/lib/libTestClass_AllDMTFArrays.so $(PACKAGE_DIR)/test
```

Comment:
@jeffaco , we only need build **one time** to get the .so files, since we need them on every Linux/Unix platform, could you help to build them to a folder that I can use them? thanks very much!
I can close this PR if you can help me build out lib libalert.so,libfilesys.so,libPresidentProvider.so,libTestClass_AllDMTFArrays.so on every platform, thanks. I only need build one time to get the libs, thank you!